web/src/routes/stall/[...slug].svelte

```diff
@@ -42,18 +38,30 @@
 </script>
 
 
-{#if $user && $user.nym === stallName || stallName === ""}
-    <ListView
+{#if $user && $user.nym === stallOwnerNym || stallOwnerNym === ""}
+    <!-- User viewing own store -->
+    <StallView
```

Comment:
I think we can include <StallView just once and simply pass the parameters as needed, no?